ui/src/lib/formikhelpers.ts

```diff
@@ -73,7 +73,7 @@ export function parseSubmitValues(values: Experiment) {
   return values
 }
 
-export function mustImmediate(formikValues: Experiment) {
+export function mustSchedule(formikValues: Experiment) {
   if (
     formikValues.target.pod_chaos.action === 'pod-kill' ||
     formikValues.target.pod_chaos.action === 'container-kill'
```

Comment:
There is another problem here. Since we only clean up the formikValues when we submit the experiment, so if we first click `Pod Chaos`, and then click `Network Chaos` and submit, then this condition `formikValues.target.pod_chaos.action === 'pod-kill'` is still satisfied, however the actual kind we are using is `NetworkChaos`. I reproduced this several times.

A possible way I can think of now is to add a check `formikValues.target.kind === 'PodChaos'`, WDYT?

Reply:
There has a solution for this is clear other `Chaos` when you select a new `Chaos`. For example, when you click the `Network Chaos` tab, the others will be reset to the default values.

What about this way?

Reply:
> A possible way I can think of now is to add a check formikValues.target.kind === 'PodChaos', WDYT? I am not sure if there is a better solution. Can we reset the experiment value each time when we move the stepper?

This way can also solve this problem. But I think if we don't clean the **old chaos** which users selected, the user will be confused when returning to the target step. Which one did he choose? There will be many chaos actions visually.

Reply:
Yes, I agree we should clear the old status. But I think it is not easy. :) Anyway please fix this using the way you like, I am new to frontend so please ignore my idea if it is wrong

Reply:
Ok, I will fix this soon.

Reply:
@yeya24 fix in 548f153 and thx for mentioned this problem.~